packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -0,0 +1,83 @@
+import { Patient } from "@metriport/core/domain/patient";
+import { out } from "@metriport/core/util/log";
+import { capture } from "@metriport/core/util/notifications";
+import { OutboundPatientDiscoveryResp } from "@metriport/ihe-gateway-sdk";
+import { errorToString } from "@metriport/shared/common/error";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { createOrUpdateCQPatientData } from "./command/cq-patient-data/create-cq-data";
+import { CQLink } from "./cq-patient-data";
+
+dayjs.extend(duration);
+
+const context = "cq.patient.discover";
+
+export async function processOutboundPatientDiscoveryResps({
+  requestId,
+  patientId,
+  cxId,
+  results,
+}: {
+  requestId: string;
+  patientId: string;
+  cxId: string;
+  results: OutboundPatientDiscoveryResp[];
+}): Promise<void> {
```

Comment:
I only noticed it now (or I would've done on the PR I pushed yesterday), but ideally we would have a type here and use the same type on the `OutboundResultPoolerDirect` to build the payload to the API.

This would make sure we're not forgetting a property in case we update this here and forget to update the caller [here](https://github.com/metriport/metriport/pull/1671/files/#diff-66df31035b6ab979bf9597ffc974f5ccf2bbf43c0df4e929389c672b85f4af65R50-R55).

Bonus if we can do the same to the other 2 lambdas. 🤩